_posts/en/newsletters/2018-12-18-newsletter.md

```diff
@@ -0,0 +1,157 @@
+---
+title: "Bitcoin Optech Newsletter #26"
+permalink: /en/newsletters/2018/12/18/
+name: 2018-12-18-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter describes the new libminisketch library for
+bandwidth efficient set reconciliation, links to an email about
+Schnorr/Taproot plans, and mentions an upcoming LN protocol
+specification meeting.  Also included are a list of notable code changes
+in the past week from popular Bitcoin infrastructure projects.
+
+## Action items
+
+- **Help test Bitcoin Core 0.17.1RC1:** the first release candidate for
+  this [maintenance release][] has been [uploaded][V0.17.1rc1].  Testing
+  by businesses and individual users of both the daemon and the GUI is
+  greatly appreciated and helps ensure the highest-quality release.
+
+## News
+
+- **Minisketch library released:** Bitcoin developers Pieter Wuille,
+  Gregory Maxwell, and Gleb Naumenko have been researching [optimized
+  transaction relay][] as described in the News section of [Newsletter
+  #9][].  One result of that research is a new standalone library they've
+  released, [libminisketch][], that allows transferring the
+  differences between two sets of information in roughly the byte size
+  of the expected differences themselves.  That may not sound
+  exciting---the `rsync` tool has done that for over two decades---but
+  libminisketch allows transferring the differences *without knowing what
+  they are ahead of time.*
+
+    For example, Alice has elements 1, 2, and 3.  Bob has elements 1 and 3.
+    Despite neither knowing which elements the other has, Alice can send
+    Bob a *sketch* the size of a single element that contains enough
+    information for him reconstruct element 2.  If Bob instead has
+    elements 1 and 2 (not 3), the exact same sketch allows him to
```

Comment:
Nit: Would changing from "exact same" to "same"  change the meaning?

Reply:
Nope, but I like the emphasis the extra word adds.

Reply:
I see :-)

Have you considered running `proselint` on the newsletters before posting the draft? I find the recommendations of `proselint` really helpful in general.

Reply:
I'd never heard of the tool before today, but it seems to be just a less featureful version of [GNU diction](https://www.gnu.org/software/diction/).  I've used that in the past, but it's not something I've run in years.  GNU style is a bit more useful, especially it's passive sentence identifier, but I don't use that routinely anymore either.

The faults identified by these tools are sometimes serious failings.  Other times, they're minor concerns or false positives that would not warrant a change by a human line editor.  I'm certainly interested in the first class of problem, but the second class of concerns are much less likely to motivate me to make revisions.  The ultimate test I use for the readability of my writing is the read-it-aloud test.  If it sounds good when I read the text to myself in a vibrant and confident voice, then I'm reasonably satisfied with the quality of my prose.

Reply:
Yes, they are similar but my experience is that `proselint` doesn't give as many false positives as `diction`. 

FWIW I think the writing in this newsletter is excellent -- so perhaps adding automation to review process is not worth it :-)

Out of curiousity: are you using any automated tools today or what does the process look like before the text hits GitHub?

Reply:
Thanks for the compliment.  The only automation I use for reviewing the newsletter is my spell checker (aspell, run in a pipe from vim) plus the broken link and broken HTML checkers in this repository's Makefile (also run by Travis).

Reply:
Nice! `mdl` is a good Markdown linter if you ever need one :-)

Reply:
Looks useful (with some configuration).  Thanks!